e2e/test/netstandard20/ProvisioningE2ETests.cs

```diff
@@ -377,42 +524,43 @@ public async Task ProvisioningDeviceClient_InvalidIdScope_Register_MqttWs_X509Gr
         [TestMethod]
         public async Task ProvisioningDeviceClient_InvalidGlobalAddress_Register_Http_Fail()
         {
-            await ProvisioningDeviceClient_InvalidGlobalAddress_Register_Fail(nameof(ProvisioningTransportHandlerHttp), nameof(SecurityProviderX509Certificate), X509EnrollmentType.Individual, null).ConfigureAwait(false);
+            await ProvisioningDeviceClient_InvalidGlobalAddress_Register_Fail(nameof(ProvisioningTransportHandlerHttp), AttestationType.x509, EnrollmentType.Individual, null, "").ConfigureAwait(false);
         }
 
         // Note: This test takes 3 minutes.
         [TestMethod]
         public async Task ProvisioningDeviceClient_InvalidGlobalAddress_Register_Amqp_Fail()
         {
-            await ProvisioningDeviceClient_InvalidGlobalAddress_Register_Fail(nameof(ProvisioningTransportHandlerAmqp), nameof(SecurityProviderX509Certificate), X509EnrollmentType.Individual, TransportFallbackType.TcpOnly).ConfigureAwait(false);
+            await ProvisioningDeviceClient_InvalidGlobalAddress_Register_Fail(nameof(ProvisioningTransportHandlerAmqp), AttestationType.x509, EnrollmentType.Individual, TransportFallbackType.TcpOnly, "").ConfigureAwait(false);
         }
 
         [TestMethod]
         public async Task ProvisioningDeviceClient_InvalidGlobalAddress_Register_AmqpWs_Fail()
         {
-            await ProvisioningDeviceClient_InvalidGlobalAddress_Register_Fail(nameof(ProvisioningTransportHandlerAmqp), nameof(SecurityProviderX509Certificate), X509EnrollmentType.Individual, TransportFallbackType.WebSocketOnly).ConfigureAwait(false);
+            await ProvisioningDeviceClient_InvalidGlobalAddress_Register_Fail(nameof(ProvisioningTransportHandlerAmqp), AttestationType.x509, EnrollmentType.Individual, TransportFallbackType.WebSocketOnly, "").ConfigureAwait(false);
         }
 
         [TestMethod]
         public async Task ProvisioningDeviceClient_InvalidGlobalAddress_Register_Mqtt_Fail()
         {
-            await ProvisioningDeviceClient_InvalidGlobalAddress_Register_Fail(nameof(ProvisioningTransportHandlerMqtt), nameof(SecurityProviderX509Certificate), X509EnrollmentType.Individual, TransportFallbackType.TcpOnly).ConfigureAwait(false);
+            await ProvisioningDeviceClient_InvalidGlobalAddress_Register_Fail(nameof(ProvisioningTransportHandlerMqtt), AttestationType.x509, EnrollmentType.Individual, TransportFallbackType.TcpOnly, "").ConfigureAwait(false);
         }
```

Comment:
I dont follow "". Looks ugly

Reply:
All 5 references to ProvisioningDeviceClient_InvalidGlobalAddress_Register_Fail are for individual attestations